Reconstruction/RecFCChhCalorimeter/tests/options/noiseLevelPerCell.py

```diff
@@ -4,28 +4,31 @@
 from Configurables import GeoSvc
 geoservice = GeoSvc("GeoSvc", detectors=[ 'file:Detector/DetFCChhBaseline1/compact/FCChh_DectEmptyMaster.xml',
                                           'file:Detector/DetFCChhECalInclined/compact/FCChh_ECalBarrel_withCryostat.xml',
-                                          'file:Detector/DetFCChhHCalTile/compact/FCChh_DetailedWedgeHCalBarrel_TileCal.xml'
+                                          'file:Detector/DetFCChhHCalTile/compact/FCChh_HCalBarrel_TileCal.xml'
                                         ],
                     OutputLevel = INFO)
 
 ecalBarrelReadoutName = "ECalBarrelPhiEta"
-hcalBarrelReadoutName = "HCalBarrelDetailedWedgeReadout"
+hcalBarrelReadoutName = "BarHCal_Readout_phieta"
 # noise files
-BarrelNoisePath = "/afs/cern.ch/work/c/cneubuse/public/FCChh/inBfield/noiseBarrel_mu100.root"
+# has HCal electr. noise levels as well as ECal electr. noise level
+BarrelNoisePath = "/afs/cern.ch/work/c/cneubuse/public/FCChh/inBfield/noiseBarrel_PU200.root"
```

Comment:
Could you copy this file to `/eos/project/f/fccsw-web/testsamples/`, change the path and perhaps comment on where to find the script to generate the noise?

Reply:
Hi Valentin,
I do not have any permissions in that directory.
Do you?

Reply:
Wrote you an email regarding the permissions, and just copied the file.

Reply:
Thanks a lot! Concerning the script for the noise. This is done for the ECal and HCal separately.. and I write this at the same time as the pileup noise is estimated. So this can only go in after pr #312 is merged.